dashboard/app/controllers/pd/professional_learning_landing_controller.rb

```diff
@@ -36,7 +31,7 @@ def index
       last_workshop_survey_course: last_enrollment_with_pending_survey.try(:workshop).try(:course),
       summarized_plc_enrollments: summarized_plc_enrollments,
       current_year_application_id: Pd::Application::TeacherApplication.find_by(user: current_user, application_year: Pd::SharedApplicationConstants::APPLICATION_CURRENT_YEAR)&.id,
-      workshops_as_participant: workshops_as_participant_data,
+      has_enrolled_in_workshop: Pd::Enrollment.for_user(current_user).any?,
```

Comment:
Now that I look at how it's being used, I'm wondering do we want to specify enrollments for the user for the current year? Or should it just be if the user has ever been enrolled before?

Reply:
oh sorry. It's a question of whether the "Getting Started" banner should show if a user has enrolled in a workshop in a previous year but not this year. I think the spec said that we shouldn't show the banner in this case

Reply:
I think if they have done any workshop we don't need to show the getting started